index.html

```diff
@@ -524,54 +526,67 @@ <h2>
     </section>
     <section>
       <h2>
-        Security and Privacy considerations
+        Security Considerations
       </h2>
       <p>
-        The Device Posture API exposes a <a>posture</a> determined from the
-        hinge angle value and other sensors.
+        No new security considerations have been reported on this specification.
       </p>
+  </section>
+    <section>
+      <h2>
+        Privacy considerations
+      </h2>
       <p>
-        Typical sensor readings are sent at a constant frequency to whomever is
-        listening to its readings. However the fold angle only communicates its
-        value when the hinge is manipulated by the user. Variations in the
-        angle’s readings and posture calculation, as well as event dispatching
-        frequency offer a possibility of fingerprinting to identify users. User
-        agents may reduce this risk by limiting or coalescing events when
-        exposing this information to web developers. Users don’t constantly
-        adjust the angle, so the fold angle value is changing in bursts: the
-        events may be dispatched at a very low frequency most of the time and
-        fire at a high frequency when the device is being opened or closed. In
-        order for the events to be dispatched, the content must be on the
-        foreground and visible to the user. Because the API does not directly
-        expose the raw angle values, but a more abstract posture with fewer
-        possible states, fingerprinting possibilities are more limited.
+        The Device Posture API exposes a <a>posture</a> determined from the hinge
+        angle sensor, but additional data can be used to determine the posture
+        such as whether the keyboard is attached or not, whether the
+        kickstand is deployed or not, etc.
       </p>
       <p>
-        If the same code using the API can be used simultaneously in different
-        window contexts on the same device it may be possible for that code to
-        correlate the user across those two contexts, creating unanticipated
-        tracking mechanisms.
+        Device Posture exposes high level concept and not sensor values
+        making fingerprinting possibilities more limited. Changing
+        posture is a physical manipulation done by the user. The frequency of
+        the posture changes are low, users don’t constantly adjust the angle,
+        and even if they do the <a>posture</a> will only be changed in certain
+        scenarios as described in [=posture values table=].
```

Comment:
Isn't the bit above about the angle changing in bursts etc relevant to this new item?

Reply:
I thought this part was generic enough and not sensor-specific:

> Users don’t constantly adjust the angle, so the fold angle value is changing in bursts: the events may be dispatched at a very low frequency most of the time and fire at a high frequency when the device is being opened or closed. In order for the events to be dispatched, the content must be on the foreground and visible to the user. Because the API does not directly expose the raw angle values, but a more abstract posture with fewer possible states, fingerprinting possibilities are more limited.

I wonder if it makes sense to incorporate part of it here.

Reply:
This doesn't really reflect the idea here. "In order for the events to be dispatched, the content must be on the foreground and visible to the user" was referring the old spec where we were dispatching raw angle values. The OS does the processing all the time regardless of the browser state, which application you're using and so forth. Then when certain conditions are met the OS will dispatch a posture change which the browser will receive and determine if it needs to be dispatched to the page (am I visible etc etc). Also, I really don't want to go to the hinge angle specifically because in a foldable PCs, the posture can change when you just undock the keyboard and the hinge angle has not changed. In terms of implementation for example we don't listen for hinge angle value from the OS but posture changes (through OS APIs like Android or alternative solutions like Windows).

Reply:
Understood!